src/ServiceControlInstaller.Engine/Instances/ServiceControlInstance.cs

```diff
@@ -101,7 +101,7 @@ public override void Reload()
             LogPath = AppConfig.Read(ServiceControlSettings.LogPath, DefaultLogPath());
             DBPath = AppConfig.Read(ServiceControlSettings.DBPath, DefaultDBPath());
             AuditQueue = AppConfig.Read(ServiceControlSettings.AuditQueue, (string)null);
-            AuditLogQueue = AppConfig.Read(ServiceControlSettings.AuditLogQueue, (string)null);
+            AuditLogQueue = AppConfig.Read(ServiceControlSettings.AuditLogQueue, string.IsNullOrEmpty(AuditQueue) ? null : $"{AuditQueue}.log");
```

Comment:
Why do we need to do this? Is it possible there are still instances that are relying on this default somewhere? If you've ever been modified by SCMU then we should have saved this setting into the file long ago